stock/app.js

```diff
@@ -56,6 +67,7 @@ app.post('/placeOrder', (request, response) => {
      orderBook.push(newOrder); 
      console.log(orderBook);
      response.send(orderBook);
+     updateStockPrice(orderType, amount, account);
```

Comment:
This should be in the post route /placeOrder Not in the process.env.  this part runs only when the server starts.  the /processOrder happens each time a user processes a buy or sell order.